ethereum/client/src/lib.rs

```diff
@@ -29,34 +32,37 @@ use crate::{
     proof::BlockMerkleRootProof,
 };
 
-pub use alloy_primitives::TxHash;
-
 mod abi;
 mod convert;
 pub mod error;
 mod proof;
 
-type ProviderType = ManagedNonceProvider<
-    Ethereum,
-    BoxTransport,
-    GasEstimatorProvider<
-        Ethereum,
-        BoxTransport,
-        SignerProvider<
-            Ethereum,
-            BoxTransport,
-            RootProvider<Ethereum, BoxTransport>,
-            EthereumSigner,
+type ProviderType = alloy::providers::fillers::FillProvider<
+    alloy::providers::fillers::JoinFill<
+        alloy::providers::fillers::JoinFill<
+            alloy::providers::fillers::JoinFill<
+                alloy::providers::fillers::JoinFill<alloy::providers::Identity, GasFiller>,
+                NonceFiller,
+            >,
+            alloy::providers::fillers::ChainIdFiller,
         >,
+        WalletFiller<alloy::network::EthereumWallet>,
     >,
+    RootProvider<alloy::transports::http::Http<alloy::transports::http::Client>>,
+    alloy::transports::http::Http<alloy::transports::http::Client>,
+    Ethereum,
 >;
 
+pub use alloy::primitives::TxHash;
```

Comment:
Move it to the other `use`s

Reply:
this is pub use used outside the crate.

Reply:
I mean move it to the line `#34`, after `use crate::...`

Reply:
Done